### PR TITLE
Refining test vectors

### PR DESCRIPTION
**Pull request by valerini**
Ready to land, the paper references test vectors from this commit (https://github.com/novifinancial/eddsa-malleability/pull/101).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,50 +85,32 @@ For a total of 15 test vectors.
...
+|TweetNaCl-js   | V | V | V | V | X | X | V | V | X | X |
+|Zebra          | V | V | V | V | V | V | X | X | X | V |
+ --------------------------------------------------------
 ```
 
 Contributors
 ------------
 
-The authors of this code are Kostas Chalkias ([@kchalkias](https://github.com/kchalkias)), François Garillot ([@huitseeker](https://github.com/huitseeker)) and Lera Valerini ([@valerini](https://github.com/valerini)).  To learn more about contributing to this project, [see this document](./CONTRIBUTING.md).
+The authors of this code are Kostas Chalkias ([@kchalkias](https://github.com/kchalkias)), François Garillot ([@huitseeker](https://github.com/huitseeker)) and Valeria Nikolaenko ([@valerini](https://github.com/valerini)).  To learn more about contributing to this project, [see this document](./CONTRIBUTING.md).
```
Comment (huitseeker) [inline README.md:113]: :facepalm: My apologies! I really had a major case of the stupids yesterday.

File: cases.json [hunk 2]
```diff
@@ -1,77 +1 @@
...
...5"},{"message":"85e241a07d148b41e47d62c63f830dc7a6851a0b1f33ae4bb2f507fb6cffec40","pub_key":"442aad9f089ad9e14647b1ef9099a1ff4798d78589e66f28eca69c11f582a623","signature":"e96f66be976d82e60150baecff9906684aebb1ef181f67a7189ac78ea23b6c0e547f7690a0e2ddcd04d87dbc3490dc19b3b3052f7ff0538cb68afb369ba3a514"},{"message":"85e241a07d148b41e47d62c63f830dc7a6851a0b1f33ae4bb2f507fb6cffec40","pub_key":"442aad9f089ad9e14647b1ef9099a1ff4798d78589e66f28eca69c11f582a623","signature":"8ce5b96c8f26d0ab6c47958c9e68b937104cd36e13c33566acd2fe8d38aa19427e71f98a473474f2f13f06f97c20d58cc3f54b8bd0d272f42b695dd7e89a8c22"},{"message":"fdaebc429f4a735932a160da1301080c13280eea8bc280d1b392c6b9e6ba3a5a","pub_key":"f7badec5b8abeaf699583992219b7b223f1df3fbbea919844e3f7c554a43dd43","signature":"edffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff7f454d370c8d9fc323a41450f8d513eafeb5b0697390c1e505a0d4ddc71f566607"},{"message":"84b698d39be126ff55fe45079e6c8bf64a0d7db6994560b4e96b7021eb39c1a1","pub_key":"f7badec5b8abeaf699583992219b7b223f1df3fbbea919844e3f7c554a43dd43","signature":"edffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff7f084d5b99c2a9463d9c8bd5026916996984eeec87ddf1d3be329006ace1b37b09"}]
```
Comment (huitseeker) [inline cases.json:1]: If we could just pass this through [`jq`](https://stedolan.github.io/jq/) for pretty-printing that would be awesome. If not I'll do it later.

File: src/main.rs [hunk 3]
```diff
@@ -930,53 +963,91 @@ pub fn non_zero_mixed_small_non_canonical() -> Result<(TestVector, TestVector)>
...
+    // #3-4: canonical S, mixed R, mixed A
     let (tv1, tv2) = non_zero_mixed_mixed().unwrap();
+    info.append(format!("| 3| ..{:} | ..{:} |  < L | mixed | mixed |    V   |    V     | succeeds unless full-order is checked |\n", &hex::encode(&tv2.message)[60..], &hex::encode(&tv2.signature)[124..]));
     vec.push(tv2); // passes cofactored, passes cofactorless
+    info.append(format!(
+        "| 4| ..{:} | ..{:} |  < L | mixed | mixed |    V   |    X     |  |\n",
+        &hex::encode(&tv1.message)[60..],
+        &hex::encode(&tv1.signature)[124..]
+    ));
     vec.push(tv1); // passes cofactored, fails cofactorless
```
Comment (huitseeker) [inline src/main.rs:1013]: The order of these two pushes ((tv2, tv1), where tv2 only passes cofactorless) is the reverse of the usual (tv1, tv2), which may induce a confusion.

**Review by huitseeker — APPROVED** [review]
Thanks a lot, that's much clearer, and since the upcoming results will only add vectors that should be ok.

File: src/main.rs (hunk 3 above)
Comment (valerini) [reply src/main.rs:1013]: Yeh, the logic here was to have vectors 0-3 to satisfy both cofactored and cofactorless.

File: cases.json (hunk 2 above)
Comment (valerini) [reply cases.json:1]: Would be great! I'll attempt in the next PR, if that's ok.

**Pull request merged**